lib/src/version_extractor.dart

```diff
@@ -0,0 +1,31 @@
+import 'dart:io';
+
+import 'package:pub_semver/pub_semver.dart';
+
+abstract class VersionExtractor {
```

Comment:
Why is it a class and why is the class abstract? A simple function would be enough.